dashboard/clojure/README.md

```diff
@@ -0,0 +1,23 @@
+## Prerequisites
+### Required
+* [Node.js](https://nodejs.org/en/) (v18.16.1)
+* [npm](https://www.npmjs.com/) (v9.5.1)
+* [Clojure](https://clojure.org/guides/getting_started) (v1.10.3)
+
+### Optional
+* [VS Code](https://code.visualstudio.com/)
+* [Calva](https://marketplace.visualstudio.com/items?itemName=betterthantomorrow.calva)
+
+## Developement Build
+### Raw terminal:
+```bash
+cd src/openllm_web_client
+npm install
+npm run dev
```

Comment:
Let's make a hatch script for this?

```bash
hatch run clojure-ui
```

?

you can add a section under hatch.toml
```toml
[envs.default.scripts]
clojure-ui = "cd dashboard/clojure && pnpm dev"
```

Reply:
lets also use pnpm here? I'm going to open a docs PR soon that utilise pnpm.